configs/nginx/timenow.local

```diff
@@ -0,0 +1,23 @@
+server {
+	listen 443; 
+	listen [::]:443;
+
+	root /var/www/timenow; 
+
+	server_name timenow.local;
+
```

Comment:
A good practice here would also be to add `\` location default handler.
For example: `try_files $uri $uri/ =404;`